### PR TITLE
Ignore stdlib ciphers

### LINKED ISSUE #572: Don't prefer the stdlib defined Ciphers
Right now urllib3 does:

```python
try:
    from ssl import _DEFAULT_CIPHERS
except ImportError:
    _DEFAULT_CIPHERS = (
        'ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+HIGH:'
        'DH+HIGH:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+HIGH:RSA+3DES:!aNULL:'
        '!eNULL:!MD5'
    )
```

Which means that it's going to prefer whatever ciphers Python prefers. However given that it's much easier to upgrade urllib3 (or requests or pip or ...) than it is to upgrade Python and that urllib3 (etc) has a much shorter release cycle I believe it would be a better idea to just ignore the ciphers that Python likes and just have urllib3 set it's own default ciphers. 

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/9627163-don-t-prefer-the-stdlib-defined-ciphers?utm_campaign=plugin&utm_content=tracker%2F192525&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F192525&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by t-8ch**
fixes #572, duplicates/supersedes #573

### CODE REVIEW

File: urllib3/contrib/pyopenssl.py [hunk 1]
```diff
@@ -85,22 +83,7 @@
...
-#
-# The general intent is:
-# - Prefer cipher suites that offer perfect forward secrecy (DHE/ECDHE),
-# - prefer ECDHE over DHE for better performance,
-# - prefer any AES-GCM over any AES-CBC for better performance and security,
-# - use 3DES as fallback which is secure but slow,
-# - disable NULL authentication, MD5 MACs and DSS for security reasons.
-DEFAULT_SSL_CIPHER_LIST = "ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:" + \
-    "ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:" + \
-    "!aNULL:!MD5:!DSS"
```
Comment (Lukasa) [inline urllib3/contrib/pyopenssl.py]: Can we keep the comment?

Comment (t-8ch) [inline urllib3/contrib/pyopenssl.py]: The list from util has some more entries:
* `ECDH-HIGH`
* `DH-HIGH`
* `RSA-HIGH`
* `!eNULL`

Comment (Lukasa) [inline urllib3/contrib/pyopenssl.py]: Heh, ok, can we rewrite the comment in the same style but in a more accurate manner?

Comment (shazow) [inline urllib3/contrib/pyopenssl.py]: Or at the very least, explain where it comes from and how to update it in the future.

Would make sense for the comment to be migrated into util.ssl_.

Comment (t-8ch) [inline urllib3/contrib/pyopenssl.py]: @shazow The comment is already migrated in the latest commit.

Comment (shazow) [inline urllib3/contrib/pyopenssl.py]: @t-8ch Has anyone ever told you that you make a great Github activity notification?

Comment (t-8ch) [inline urllib3/contrib/pyopenssl.py]: Nope, and I am not sure how to parse this :-)

Comment (shazow) [inline urllib3/contrib/pyopenssl.py]: It's a backhanded way of me whining that noticing commit changes in PRs is really hard on Github. :P

**Pull request merged**